renku/models/_jsonld.py

```diff
@@ -220,7 +224,8 @@ def asjsonld(
     def convert_value(v):
         """Convert special types."""
```

Comment:
What is a special type?

Reply:
non-standard 😉

Reply:
Can you put a little bit less ambiguous comment? :hugs: